src/components/Web3Status/index.tsx

```diff
@@ -107,6 +108,22 @@ const Web3StatusConnected = styled(Web3StatusGeneric)<{ pending?: boolean }>`
           pending ? darken(0.1, theme.deprecated_primary1) : darken(0.1, theme.deprecated_bg2)};
     }
   }
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.lg}px`}) {
+    width: ${({ isNftActive, pending }) => isNftActive && !pending && '44px'};
+    height: ${({ isNftActive, pending }) => isNftActive && !pending && '44px'};
+    div {
```

Comment:
don't love the inner div selector here, can you just make the other thing a styled component?